packages/nc-gui/components/smartsheet/PlainCell.vue

```diff
@@ -10,19 +10,7 @@ import {
   timeFormats,
 } from 'nocodb-sdk'
 import dayjs from 'dayjs'
-import {
-  computed,
-  isBoolean,
-  isDate,
-  isDateTime,
-  isInt,
-  parseProp,
-  ref,
-  storeToRefs,
-  useAttachment,
-  useBase,
-  useMetas,
-} from '#imports'
+import { computed, isBoolean, isDate, isDateTime, isInt, parseProp, ref, storeToRefs, useBase, useMetas } from '#imports'
```

Comment:
I removed `const { getPossibleAttachmentSrc } = useAttachment()` and `useAttachment` from import because it is unused and  `useMetas` is still their, I have not removed `useMetas`